src/components/Recaptcha.jsx

```diff
@@ -5,11 +5,11 @@ function Recaptcha({ onToken }) {
 
   useEffect(() => {
     const script = document.createElement("script");
-    script.src = "https://www.google.com/recaptcha/api.js?render=6LdgNhIqAAAAAHaQlxrKbWOSCh79HjlWz8JfCEOE";
+    script.src = "https://www.google.com/recaptcha/api.js?render=6LeMShEqAAAAANL4rT2yerYM70w-yM_HPi4TvrEP";
     script.addEventListener("load", () => {
       window.grecaptcha.ready(() => {
         window.grecaptcha
-          .execute("6LdgNhIqAAAAAHaQlxrKbWOSCh79HjlWz8JfCEOE", { action: "submit" })
+          .execute("6LeMShEqAAAAANL4rT2yerYM70w-yM_HPi4TvrEP", { action: "submit" })
```

Comment:
Add the key to an env file and use it. This is a key that should not be exposed to everyone